src/coreComponents/dataRepository/Group.hpp

```diff
@@ -1207,7 +1207,7 @@ class Group
    * @note An error will be raised if wrapper does not exist or type cast is invalid.
    */
   template< typename T, typename LOOKUP_TYPE >
-  traits::ViewTypeConst< T >
+  GEOSX_DECLTYPE_AUTO_RETURN
   getReference( LOOKUP_TYPE const & lookup ) const
```

Comment:
Did we discuss changing the name of `getReference` to something more appropriate? Did we leave it at "lets not change it now because the view plays the role of a reference"?